app/controllers/lti_deployment_controller.rb

```diff
@@ -175,6 +186,15 @@ def create_course
     redirect_to edit_course_path(new_course)
   end
 
+  def create_lti_grades
+    assessment = Assessment.find(params[:assessment_id])
```

Comment:
If this path requires an assessment_id then nest the route under "assessments". 

See my comment on routes.rb

Reply:
Per discussion this will be a separate PR